README.md

```diff
@@ -1,2 +1,34 @@
 # scmd-narcolepsy
-Investigation of medicines used in secondary care to treat excess daytime sleepiness caused by narcolepsy
+Investigation of medicines used in Primary (English Prescribing Data) care & secondary (Secondary Care Medicines Data) care to treat narcolepsy
+
+Description of R scripts:
+  - "OctGitHubFinal.R" - Data cleaning + analysis of secondary care data (SCMD). Largely adapated from initial code provided by DataLab team
+  - "OctGitMapping.R" - Incoporation of SCMD with geographic data to plot volumes of DDDs per ICB on map of ICBs + using separate data frames
+  - "OctGitPrimaryCare.R" - Data Cleaning + creating maps for primary care (English prescribing data) mapped to ICB regions
+  - "OctGitHubPrimaryReformatCode.R" - Data Cleaning/Reformatting primary care data to facilitate create equivalent plots to secondary care plots (National 3 monthly rolling average, volumes per geographic region, Top ICBs prescribing)
+
+Description of Datasets:
+Raw data (.csv):
+  - "epdall" - Primary care ("English Prescribing Data") data for all prescriptions of Sodium Oxybate, Pitolisant, Solriamfetol issued between Jan 2019 - Dec 2022. Excluding prescriptions which could not be mapped to ICBs due to incorrect codes
```

Comment:
could you add the source of the data somewhere, maybe a link where you downloaded the datasets in this list?